packages/manager/apps/pci-block-storage/src/api/select/catalog.ts

```diff
@@ -21,17 +21,22 @@ export type TModelName = Readonly<{
 
 type TVolumeModelWithName<T extends TVolumeAddon> = T & TModelName;
 
+const is3az = (catalogRegions: TRegion[], region: string) =>
+  catalogRegions.find((r) => r.name === region).type === 'region-3-az';
```

Comment:
```suggestion
  catalogRegions.find((r) => r.name === region)?.type === 'region-3-az';
```
find can return `undefined` no ?